### ISSUE TITLE
Doc: Examples

### ISSUE THREAD

**Issue by vandres** [vasani-arpit/cordova-plugin-GoogleDriveAppDataSync#1]
Hi, could you add an example how to connect, create a directory, create a file, read all files? Something like that

**Comment by vasani-arpit** [comment]
I really like to do that but I don't remember steps I followed to make API keys for this type of project.

I will try to retrace my steps and will update here.

**Comment by vandres** [comment]
Thanks a lot!

**Comment by vandres** [comment]
Looks like the API will be shut down soon: https://developers.google.com/drive/android/deprecation Can you confirm that? :(

**Comment by vasani-arpit** [comment]
No. I don't think that is the case. I see this feature in many of the famous apps(like Whatsapp). I think the G-Suite support for this API may go down.

**Comment by vandres** [comment]
Yeah, but maybe they use the REST API
